app/lib/onboarding/sign_up/pages/new_privacy_policy_page.dart

```diff
@@ -0,0 +1,1001 @@
+    // return anchorsController
+    //     .getIndexedAnchors()
+    //     .map(_toDocumentSection)
+    //     .toList();
+  }
+
+  void scrollToSection(String documentSectionId) {
+    anchorsController.scrollToAnchor(documentSectionId);
+  }
+
+  @override
+  void dispose() {
+    documentSections.close();
+  }
+}
+
+class NewPrivacyPolicy extends StatelessWidget {
+  NewPrivacyPolicy({Key? key}) : super(key: key) {
+    _itemScrollController = ItemScrollController();
+    _itemPositionsListener = ItemPositionsListener.create();
+    _anchorsController = AnchorsController();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    // Temporary - for development on Windows on Android Tablet
+    SystemChrome.setEnabledSystemUIMode(SystemUiMode.immersive);
+    return BlocProvider(
+      bloc: PrivacyPolicyBloc(_anchorsController),
+      child: Theme(
+        data: Theme.of(context).copyWith(
+            floatingActionButtonTheme: FloatingActionButtonThemeData(
+          backgroundColor: Theme.of(context).primaryColor,
+        )),
+        child: Builder(builder: (context) {
+          return Scaffold(
+            body: Stack(
+              children: [
+                Align(
+                  child: _DarkLightModeToggle(),
+                  alignment: Alignment.bottomRight,
+                ),
+                Center(
+                  child: Row(
+                    children: [
+                      _TableOfContents(),
+                      VerticalDivider(),
+                      Flexible(
+                        child: ConstrainedBox(
+                          constraints: BoxConstraints(maxWidth: 600),
+                          child: Padding(
+                            padding: const EdgeInsets.only(top: 20),
+                            child: Column(
+                              mainAxisAlignment: MainAxisAlignment.center,
+                              crossAxisAlignment: CrossAxisAlignment.center,
+                              children: const [
+                                _PrivacyPolicyHeading(),
+                                Padding(
+                                  padding: const EdgeInsets.symmetric(
+                                      horizontal: 20, vertical: 8.0),
+                                  child: _PrivacyPolicySubheading(),
+                                ),
+                                Divider(),
+                                Flexible(child: _PrivacyPolicyMarkdown()),
+                                Divider(),
+                                Padding(
+                                  padding:
+                                      const EdgeInsets.symmetric(vertical: 20),
+                                  child: _AcceptionButtons(),
+                                )
+                              ],
+                            ),
+                          ),
+                        ),
+                      ),
+                    ],
```

Comment:
@nilsreichardt 
I want to constrain the privacy policy text to a maxWidth (wich works via `ConstrainedBox `) but I want to center the text (which does not work).       
Wrapping the `Flexible` in a `Center` or `Alignement` does not work. Changing the `mainAxisAlignement` of the `Row` which wrapps these widgets isn't a soluton (i think), since I want to lay out the table of contents at the left and only center the text.

![image](https://user-images.githubusercontent.com/29028262/166310383-68804c77-354d-473c-9936-0efd3ae1c1af.png)

Reply:
You need to wrap the `ConstrainedBox` into a `Expanded` widget. Therefore, takes the widget the complete remaining space of the parent `Row` (`_TableOfContents` and `VerticalDivider` takes only what the need). Now to center the `ConstrainedBox` inside the `Expanded` widget, you need to wrap it with a `Center` widget.

```dart
Row(
  children: [
    _TableOfContents(),
    VerticalDivider(),
    Expanded(
      child: Center(
        child: ConstrainedBox(
          constraints: BoxConstraints(maxWidth: 600),
          child: Padding(
            padding: const EdgeInsets.only(top: 20),
            child: Column(
              mainAxisAlignment: MainAxisAlignment.center,
              crossAxisAlignment: CrossAxisAlignment.center,
              children: const [
                _PrivacyPolicyHeading(),
                Padding(
                  padding: const EdgeInsets.symmetric(
                      horizontal: 20, vertical: 8.0),
                  child: _PrivacyPolicySubheading(),
                ),
                Divider(),
                Flexible(child: _PrivacyPolicyMarkdown()),
                Divider(),
                Padding(
                  padding: const EdgeInsets.symmetric(
                      vertical: 20),
                  child: _AcceptionButtons(),
                )
              ],
            ),
          ),
        ),
      ),
    ),
  ],
```

![image](https://user-images.githubusercontent.com/24459435/166314107-0a1850f4-2eeb-4ab1-8177-6c73d3edd666.png)

Reply:
Wtf, I'm sure I tried this exact combination before and for me if I used `Expanded` my `BoxConstraints` were just ignored 🤔🤔